src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IVariableDeclaration.vb

```diff
@@ -0,0 +1,1084 @@
+    End Sub
+
+    Function ReturnInt() As Integer
+        Return 1
+    End Function
+End Module
+    ]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableDeclarationStatement (2 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: i1 As System.Int32 (OperationKind.VariableDeclaration)
+    Initializer: IInvocationExpression (static Function Program.ReturnInt() As System.Int32) (OperationKind.InvocationExpression, Type: System.Int32)
+  IVariableDeclaration: i2 As System.Int32 (OperationKind.VariableDeclaration)
+    Initializer: IInvocationExpression (static Function Program.ReturnInt() As System.Int32) (OperationKind.InvocationExpression, Type: System.Int32)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree)
+        End Sub
+
+        <Fact(), WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")>
+        Public Sub DimAsNew()
+            Dim source = <![CDATA[
+Module Program
+    Class C
+    End Class
+    Sub Main(args As String())
+        Dim p1 As New C'BIND:"Dim p1 As New C"
+    End Sub
+End Module
+    ]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableDeclarationStatement (1 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: p1 As Program.C (OperationKind.VariableDeclaration)
+    Initializer: IObjectCreationExpression (Constructor: Sub Program.C..ctor()) (OperationKind.ObjectCreationExpression, Type: Program.C)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree)
+        End Sub
+
+        <Fact(), WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")>
+        Public Sub MultipleDimAsNew()
+            Dim source = <![CDATA[
+Module Program
+    Sub Main(args As String())
+        Dim i1, i2 As New Integer'BIND:"Dim i1, i2 As New Integer"
+    End Sub
+End Module
+    ]]>.Value
+
+            Dim expectedOperationTree = <![CDATA[
+IVariableDeclarationStatement (2 variables) (OperationKind.VariableDeclarationStatement)
+  IVariableDeclaration: i1 As System.Int32 (OperationKind.VariableDeclaration)
+    Initializer: IObjectCreationExpression (Constructor: Sub System.Int32..ctor()) (OperationKind.ObjectCreationExpression, Type: System.Int32)
+  IVariableDeclaration: i2 As System.Int32 (OperationKind.VariableDeclaration)
+    Initializer: IObjectCreationExpression (Constructor: Sub System.Int32..ctor()) (OperationKind.ObjectCreationExpression, Type: System.Int32)
+]]>.Value
+
+            VerifyOperationTreeForTest(Of LocalDeclarationStatementSyntax)(source, expectedOperationTree)
+        End Sub
+
+
+        <Fact(), WorkItem(17599, "https://github.com/dotnet/roslyn/issues/17599")>
+        Public Sub DimAsNewNoObject()
+            Dim source = <![CDATA[
+Module Program
+    Sub Main(args As String())
+        Dim i1 As New'BIND:"Dim i1 As New"
```

Comment:
>As New [](start = 15, length = 6)

Was this scenario added in response to my feedback? I probably was too brief. Saying "As New" form, I meant "As New <type name> ...", i.e. As New clause, see AsNewClauseSyntax in Syntax.xml